mission1/login/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>판다마켓 : 로그인</title>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE-edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" href="../img/panda.png">
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <div class="form-div">
+      <form>
+        <!-- logo -->
+        <a class="logo" href="/">
+          <img class="logo-img" src="../img/logo_size_01.png" alt="판다마켓">
+        </a>
+        <!-- email -->
+        <label for="input-email">이메일</label>
+        <input id="input-email" name="email" type="email" placeholder="이메일을 입력해주세요" autocomplete="email" required>
```

Comment:
```suggestion
        <label for="email">이메일</label>
        <input id="email" name="email" type="email" placeholder="이메일을 입력해주세요" autocomplete="email" required>
```

label의 for은 input 요소의 상태에 따라 함께 라벨 접근성이 적용되도록 해주는 좋은 기능입니다.
다만, 누가봐도 email input을 위한 label에게 굳이 `input-email` 이라고 적용하는건 반복적인 내용으로 사료되어요.
따라서 불필요한 input이라는 내용을 뺴고 적용하면 더 좋지 않을까요?